index.html

```diff
@@ -1,215 +1,88 @@
-<!DOCTYPE HTML>
-<html>
-
-
-
-<head>
-
-    <title> TimberBase </title>
-    <link rel="stylesheet" type="text/css" href="style.css">
-</head>
-
-
-<header>
-
-
-    <div class="main">
-
-        <div class="logo">
-            <img src="../Assets/logo.png">
-        </div>
-
-
-
-        <ul class="mainmenu">
-
-            <li> <a href="#catalog">Catalog</a></li>
-            <li> <a href="#contact_link">Contact</a></li>
-            <li> <a href="#galerie">Galerie</a></li>
-            <li> <a href="#video">Video</a></li>
-
-        </ul>
-
-    </div>
-</header>
-
-
-<body>
-    <div class="body_1">
-        <h1 class="title">
-            TIMBERBASE
-        </h1>
-        <h2 class="description">
-            Lemn Tratat, Lambriu, Decking, Placaje .... totul prelucrat și întreținut la standarde Europene!
-        </h2>
-
-        <button class="cta_button"> Catalog </button>
-
-    </div>
-
-    <div class="body_2">
-        <a name="catalog"> </a>

```

Comment:
<img src="../Assets/product.jpeg" alt="product" class="box_img">
                <h3 class="boxh3"> Placaje</h3>
                <dev class="box_button" onselectstart="return false">Detalii</dev>


Modify this